targets/avr.ld

```diff
@@ -5,7 +5,7 @@ MEMORY
     RAM (xrw)       : ORIGIN = 0x800000 + __ram_start, LENGTH = __ram_size
 }
 
-ENTRY(__vector_RESET)
+ENTRY(main)
```

Comment:
This change is necessary because of a quirk in LTO. This way, `main` isn't optimized away (which we certainly don't want!).
In any case, the entry symbol isn't really relevant here. It's just a way to make absolutely sure the given symbol isn't removed by the linker.

Reply:
Thanks for elaborating on this.